dockerfiles/start.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -ex
+if [ -n "${S3_MODEL_DIR}" ]; then
+  aws s3 sync --exclude '*/dataset/*' --exclude '*/cache/*' --exclude 'iteration_*.pth' --exclude '*_optim.pth' "${S3_MODEL_DIR}" ./model
+  genienlp kfserver --path ./model $*
+else
+  genienlp $*
+fi
```

Comment:
I wonder if we can avoid the start script with an init container instead. As this stands, depending on the environment variable it's a bit opaque to know if the docker image behaves normally or it behaves like the kfserver.

Also, you'll want to use `"$@"` instead of `$*`

Reply:
I'll look into if it's easy to add init container to the inference service spec.  If not, we can also break out kfserver into a separate Dockerfile.

Reply:
Looks like the spec only allows one container: https://github.com/kubeflow/kfserving/blob/master/python/kfserving/docs/V1alpha2CustomSpec.md